dune

```diff
@@ -0,0 +1,11 @@
+(library
+(name lzo)
+ (public_name lzo)
+ (flags -linkall)
+ (wrapped false)
+ (libraries bytes)
+ (foreign_stubs
+   (language c)
+   (names lzo_stubs)
+   (include_dirs /opt/homebrew/Cellar/lzo/2.10/include/)
+   (flags :standard -llzo2 -L/opt/homebrew/Cellar/lzo/2.10/lib/)))
```

Comment:
Using the result of `pkg-config` here is preferred.

Reply:
Yeah, I figured, this is why I asked how to do that in https://github.com/ocaml/dune/issues/8412 - I wasn't able to find anything useful in the documentation. `pkg-config` is mentioned only in `ctypes` Dune plugin but this doesn't apply here.